ledger/src/blockstore.rs

```diff
@@ -1149,6 +1149,43 @@ impl Blockstore {
         conflicting_shred
     }
 
+    /**
+     * Insert the data shred into the write batch and update its in-memory
+     * copy of the associated meta-data.
+     *
+     * Note that this function currently must be invoked while holding
+     * the insert_shreds_lock as it performs read-modify-write operations
+     * on multiple column families.
+     *
+     * The column families which it may directly update are cf::DeadSlots
```

Comment:
Similar comment as other PR - maybe mention that changes to these CF's would be added to `write_batch`